### PR TITLE
Doc review

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**
#### Descriptions:
Review release notes

#### Fix:
_Add fix/changes here_

### CODE REVIEW

File: release_notes.md [hunk 1]
```diff
@@ -1,55 +1,84 @@
...
+
+## Playbook Enhancements
+
+- A new playbook **Set Asset Icon** under the collection **02 - Use Case - IT OT Asset Management** correlates *Asset Icon* with *Asset* as per the asset's category.
+
+- The playbook *Set Assets With MEF Tag* has been renamed to **Tag as Most Essential Function (MEF)**.
+    - A manual input now asks for a reason when tagging assets as MEF.
+
+- Renamed the playbook *Links MITRE Matrices To Alert* to **Links Assets To Alert**
+    - Removed the correlation with MITRE module data as the correlation is now a part of **MITRE ATT&CK Enrichment Framework** v2.2.0 and later.
```
Comment (cs-ankit-gupta) [inline release_notes.md]: Can we provide MITRE SP v2.2.0 doc link over here

File: release_notes.md [hunk 2]
```diff
@@ -1,55 +1,84 @@
...
\ No newline at end of file
+- Dashboards **Asset Change Activity Tracking**, **Asset Overview**, and **Asset Risk Overview** can now be assigned to various roles; the role assignment was not possible earlier.
+
+- Following playbook collection names were changed to be inline with other playbook collections in FortiSOAR&trade;:
+    - **00 - Use Case - Asset Change Activity** renamed to **02 - Use Case - Asset Change Activity**.
+    - **00 - Use Case - Asset Management** renamed to **02 - Use Case - IT OT Asset Management**.
+
+## Deprecations
+
+- Following fields from the **Asset** module in this solution pack are now a part of **SOAR Framework** v2.2.0 and later:
```
Comment (cs-ankit-gupta) [inline release_notes.md]: Same as above - Hyperlink

File: release_notes.md [hunk 3]
```diff
@@ -1,55 +1,84 @@
...
+    - Printer
+
+## Dashboard Enhancements
+
+- The **Asset Risk Overview** dashboard now appears with following enhancements:
+
+    - A section lists **Top 10 Devices With Vulnerabilities**
+
+    - A section **Recent Alerts On Critical Assets** comes with pre-defined filter to list assets based on their criticalities &ndash; *Critical* or *Very Critical*
+
```
Comment (cs-dev-chaudhary) [inline release_notes.md:31]: Can we call 'widget' instead of 'section' ?

File: release_notes.md [hunk 4]
```diff
@@ -1,55 +1,84 @@
...
+    | Asset                     | ManyToMany  |
+    | Protocol                  | Picklist    |
+    | ESPZone                   | Text Fields |
+    | Facility                  | Text Fields |
+    | Level                     | Picklist    |
+    | Zone                      | Picklist    |
+    | Asset Type                | Picklist    |
+    | Vulnerability Risk Status | Picklist    |
+
+- Following picklists from the **Asset** module in this solution pack are now a part of **SOAR Framework** v2.2.0 and later:
```
Comment (cs-ankit-gupta) [inline release_notes.md]: Same as above - Hyperlink

File: release_notes.md [hunk 5]
```diff
@@ -1,55 +1,84 @@
...
+    - `AssetLevel`
+    - `AssetProtocol`
+    - `AssetType`
+    - `AssetZone`
+    - `VulnerabilityRiskStatus`
+
+- The playbook *MITRE ATT&CK > Fetch Latest Data* from the collection **02 - Use Case - IT OT Asset Management** is now deprecated as the MITRE data can be ingested using data ingestion from MITRE connector.
+
+- The playbook **Create Alerts Record** is now deprecated
+- Following playbooks are now deprecated as they are a part of **SOAR Framework** v2.2.0 and later:
```
Comment (cs-ankit-gupta) [inline release_notes.md]: Same as above - Hyperlink

File: release_notes.md [hunk 6]
```diff
@@ -1,55 +1,84 @@
...
+
+## Dashboard Enhancements
+
+- The **Asset Risk Overview** dashboard now appears with following enhancements:
+
+    - A section lists **Top 10 Devices With Vulnerabilities**
+
+    - A section **Recent Alerts On Critical Assets** comes with pre-defined filter to list assets based on their criticalities &ndash; *Critical* or *Very Critical*
+
+- The **Asset Overview** dashboard displays *Asset Hostname* as the asset's title
```
Comment (cs-dev-chaudhary) [inline release_notes.md]: The **Asset Overview** dashboard displays *Asset Hostname* as the asset's title under Record Distribution widget named "Critical Asset Distribution".

File: release_notes.md [hunk 7]
```diff
@@ -1,55 +1,84 @@
...
+    - Changed the widget name from *MEF Asset* to **Most Essential Function (MEF) Assets** to better describe the asset listing
```
Comment (cs-dev-chaudhary) [inline release_notes.md]: In file view, above two are shown as parent & child relation whereas they are two independent widgets/points.

File: release_notes.md [hunk 8]
```diff
@@ -1,55 +1,84 @@
...
-    - AssetZone
-    - VulnerabilityRiskStatus
-
-- Removed playbook `MITRE ATT&CK > Fetch Latest Data` from collection `02 - Use Case - IT OT Asset Management` as MITRE data can be ingested using data ingestion from MITRE connector.
-- Removed playbook `Create Alerts Record`
-- Removed playbook `Alert - Escalate To Incident`, `Alert - Escalate to Incident (Link Relations)`, and `Alert - Escalate To Incident (No Trigger)`, as implemented it in **Soar Framework Solution Pack v2.1.1** 
-- Moved `MITRE Widget` from dashboard **Asset Risk Overview** to **MITRE ATT&CK Enrichment Framework v2.2.0**
\ No newline at end of file
+- Dashboards **Asset Change Activity Tracking**, **Asset Overview**, and **Asset Risk Overview** can now be assigned to various roles; the role assignment was not possible earlier.
+
```
Comment (cs-ankit-gupta) [inline release_notes.md:47]: Point should be
Dashboards **Asset Change Activity Tracking**, **Asset Overview**, and **Asset Risk Overview** is now assigned to various roles; the role assignment was not present earlier.

File: release_notes.md [hunk 9]
```diff
@@ -1,55 +1,84 @@
...
+- Following fields from the **Asset** module in this solution pack are now a part of **SOAR Framework** v2.2.0 and later:
+
+    | Field Name                | Input Type  |
+    |:--------------------------|:------------|
+    | Network                   | Text Field  |
+    | Subnet                    | Text Field  |
+    | Firmware                  | Text Field  |
+    | Vendor                    | Text Field  |
+    | Product                   | Text Field  |
```
Comment (cs-ankit-gupta) [inline release_notes.md]: We need to remove field 'Product'. As this was not part of OT Asset Management v1.0.0

File: release_notes.md [hunk 10]
```diff
@@ -1,55 +1,84 @@
...
+
+- Following picklists from the **Asset** module in this solution pack are now a part of **SOAR Framework** v2.2.0 and later:
+    - `AssetLevel`
+    - `AssetProtocol`
+    - `AssetType`
+    - `AssetZone`
+    - `VulnerabilityRiskStatus`
+
+- The playbook *MITRE ATT&CK > Fetch Latest Data* from the collection **02 - Use Case - IT OT Asset Management** is now deprecated as the MITRE data can be ingested using data ingestion from MITRE connector.
+
```
Comment (cs-dev-chaudhary) [inline release_notes.md:81]: Connector is to be renamed from "MITRE" to "MITRE ATT&CK"

File: release_notes.md (hunk 5 above)
Comment (cs-ankit-gupta) [reply release_notes.md]: SOAR Framework v2.1.1

File: release_notes.md (hunk 3 above)
Comment (cs-abhishek-shukla) [reply release_notes.md:31]: **Recent Alerts on Critical Assets** is a card tile widget. Calling it something else may trigger users to search for that widget on Content Hub

File: release_notes.md (hunk 1 above)
Comment (cs-abhishek-shukla) [reply release_notes.md]: I will have the link point to Content Hub

File: release_notes.md (hunk 6 above)
Comment (cs-abhishek-shukla) [reply release_notes.md]: Fixed.

File: release_notes.md (hunk 7 above)
Comment (cs-abhishek-shukla) [reply release_notes.md]: Fixed

File: release_notes.md (hunk 10 above)
Comment (cs-abhishek-shukla) [reply release_notes.md:81]: Fixed

File: release_notes.md [hunk 11]
```diff
@@ -1,55 +1,85 @@
...
+- The **Asset Overview** dashboard displays *Asset Hostname* as the asset's title under **Critical Asset Distribution** *Record Distribution* widget.
+
+- Changed the widget name from *MEF Asset* to **Most Essential Function (MEF) Assets** to better describe the asset listing
   
+## Other Enhancements
+
+- The tag names `SampleAsset` and `SampleAlert` have been renamed to `Sample` to simplify alert listing using tags.
+
+- A new connector **Qualys** now installs with this solution pack.
```
Comment (cs-ankit-gupta) [inline release_notes.md]: This is bug (of not installing Qualys connector). So it should be part of bug fixes

**Review by cs-ankit-gupta — CHANGES_REQUESTED** [review]

File: release_notes.md (hunk 2 above)
Comment (cs-abhishek-shukla) [reply release_notes.md]: Fixed

File: release_notes.md (hunk 4 above)
Comment (cs-abhishek-shukla) [reply release_notes.md]: Fixed

File: release_notes.md (hunk 5 above)
Comment (cs-abhishek-shukla) [reply release_notes.md]: Fixed

File: release_notes.md (hunk 8 above)
Comment (cs-abhishek-shukla) [reply release_notes.md:47]: Fixed

File: release_notes.md (hunk 9 above)
Comment (cs-abhishek-shukla) [reply release_notes.md]: Fixed

File: release_notes.md (hunk 11 above)
Comment (cs-abhishek-shukla) [reply release_notes.md]: Fixed

**Review by cs-dev-chaudhary — APPROVED** [review]

File: release_notes.md [hunk 12]
```diff
@@ -1,55 +1,85 @@
...
-    - AssetZone
-    - VulnerabilityRiskStatus
-
-- Removed playbook `MITRE ATT&CK > Fetch Latest Data` from collection `02 - Use Case - IT OT Asset Management` as MITRE data can be ingested using data ingestion from MITRE connector.
-- Removed playbook `Create Alerts Record`
-- Removed playbook `Alert - Escalate To Incident`, `Alert - Escalate to Incident (Link Relations)`, and `Alert - Escalate To Incident (No Trigger)`, as implemented it in **Soar Framework Solution Pack v2.1.1** 
-- Moved `MITRE Widget` from dashboard **Asset Risk Overview** to **MITRE ATT&CK Enrichment Framework v2.2.0**
\ No newline at end of file
+- Dashboards **Asset Change Activity Tracking**, **Asset Overview**, and **Asset Risk Overview** now come pre-assigned to relevant roles.
+
```
Comment (cs-ankit-gupta) [inline release_notes.md:47]: Please add these Role name: 
1. Full App Permission
2. SOC Manager
3. SOC Analyst

File: docs/contents.md [hunk 13]
```diff
@@ -64,8 +64,8 @@ The **OT - Asset Management** solution pack contains the following resources.
 | :------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | OT - Add Sample Assets                       | This scenario adds 86 different samples of OT Assets (different Purdue Levels, Types, etc.) for testing dashboards, use case playbooks, reports, and other actions. |
```
Comment (cs-ankit-gupta) [inline docs/contents.md]: 87 different sample.....

File: docs/usage.md [hunk 14]
```diff
@@ -5,77 +5,111 @@
...
-## Simulation mode
+## Simulations
 
 The simulation mode has some sample data that helps you get a better understanding of how the solution pack functions. Following steps help you use the solution pack with some included sample data.
 
-1. **OT - Add Sample Assets**
+### OT - Add Sample Assets
 
-    - Browse to `Simulations` > `OT - Add Sample Assets` scenario and click **Simulate Scenario**.
+This scenario generates 86 OT Assets of different Levels, Types, and other categories as an example for testing dashboards, use case playbooks, reports and other actions.
```
Comment (cs-ankit-gupta) [inline docs/usage.md]: 87 OT Assets

File: docs/usage.md [hunk 15]
```diff
@@ -5,77 +5,111 @@
...
+This scenario generates following alerts, each at an interval of 2 seconds:
 
-    - Escalate the alert to an incident by checking `Select All` and `Include this record` then running the playbook **Escalate**. Please provide all incident details. (An incident is created with the link available in the alert's comment)
+- Stuxnet peer-to-peer communication attempt
+- Anomalous communication between two Windows XP machines
+- Communication between public and private networks
+- STEP7 configuration download command
 
-    - Incident get correlated with all the **IOCs**, **Alerts**, **Assets**, **Technique**, **Mitigation**, and **Software**.
+<table>
```
Comment (cs-ankit-gupta) [inline docs/usage.md]: Please add this Demo Mode after "Simulation" heading

File: docs/usage.md [hunk 16]
```diff
@@ -5,77 +5,111 @@
...
+    - ThinkCentre M910s-1181
+    - HP - PB7181
+
+- Each alert is associated with assets based on the assets' source and destination IP.
+
+- The **Correlations** tab of an alert links *Group*, *Tactics*, *Technique*, *Sub-Technique*, and *Software*.
+
+- Possible mitigation actions are listed in the alert's **Recommended ATT&CK Mitigation** tab.
+
+- After enrichment completes, the **Stuxnet peer-to-peer communication attempt** alert contains similar alerts in the **Workspace** > **Recommendations** tab.
```
Comment (cs-ankit-gupta) [inline docs/usage.md]: ........alerts in the **Workspace** > **Recommendations** > **Records** tab.

File: docs/usage.md [hunk 17]
```diff
@@ -5,77 +5,111 @@
...
+- Possible mitigation actions are listed in the alert's **Recommended ATT&CK Mitigation** tab.
+
+- After enrichment completes, the **Stuxnet peer-to-peer communication attempt** alert contains similar alerts in the **Workspace** > **Recommendations** tab. 
+
+    - You can escalate this alert to an incident by selecting **Select All** and **Include this record** checkboxes, click the **Playbook** drop-down and select **Escalate**.
     
-    - Playbooks in the War Room can be used to take action, such as `Isolate Devices From Network`, `Scan All Assets Involved`, and `Update Firewall Policy`.
+    - On the **Escalate** pop-up window, enter relevant details and click the button **Escalate**. An incident is created with the link available in the alert's comment.
+
+- Incidents are correlated with all the *Indicators*, *Alerts*, *Assets*, *Technique*, *Mitigation*, and *Software*.
```
Comment (cs-ankit-gupta) [inline docs/usage.md]: Incidents are correlated with all the *Indicators*, *Alerts*, *Assets*, MITRE ATT&CK's **Group**, **Tactics**, **Techniques**, **Sub-techniques**, **Mitigations**, **Software**, and **Mitigation**

File: release_notes.md (hunk 12 above)
Comment (cs-abhishek-shukla) [reply release_notes.md:47]: Fixed

File: docs/contents.md (hunk 13 above)
Comment (cs-abhishek-shukla) [reply docs/contents.md]: Fixed

File: docs/usage.md (hunk 14 above)
Comment (cs-abhishek-shukla) [reply docs/usage.md]: Fixed

File: docs/usage.md (hunk 15 above)
Comment (cs-abhishek-shukla) [reply docs/usage.md]: Fixed

File: docs/usage.md (hunk 16 above)
Comment (cs-abhishek-shukla) [reply docs/usage.md]: Fixed

File: docs/usage.md [hunk 18]
```diff
@@ -5,90 +5,197 @@
...
+- [OT - Add Sample Assets](#ot---add-sample-assets)
 
-# OT- Asset Management
+- [OT - Add Sample Alerts](#ot---add-sample-alerts)
 
-### Step to be followed after installing SP:
+- [OT - Stuxnet Attack Scenario](#ot---stuxnet-attack-scenario)
 
-1. Ingest MITRE **Group**, **Tactics**, **Techniques**, **Sub-techniques**, **Mitigations**, **Software**, and **Mitigation** data using `MITRE ATT&CK` connector on `ICS` configuration.
+- [OT - Asset Change Activity Scenario](#ot---add-sample-asset-change-activity-record)
```
Comment (cs-ankit-gupta) [inline docs/usage.md]: link is broken cc: @cs-abhishek-shukla

File: README.md [hunk 19]
```diff
@@ -1,18 +1,56 @@
...
 
-* **Version**: 2.0.0
-* **Certified**: Yes
-* **Publisher**: Fortinet
-* **Compatible Version**: Minimum compatible FortiSOAR v7.4.1
-* [Release Notes](./release_notes.md)
+- **Version**: 2.0.0
+- **Certified**: Yes
+- **Publisher**: Fortinet
+- **Compatible Version**: Minimum compatible FortiSOAR v7.4.1
```
Comment (cs-ankit-gupta) [inline README.md]: Can we make it "Minimum FortiSOAR v7.4.1" or "Minimum v7.4.1"

File: release_notes.md [hunk 20]
```diff
@@ -1,55 +1,89 @@
...
+    - A widget titled **Recent Alerts On Critical Assets** now comes with pre-defined filter to list assets based on their criticalities &ndash; *Critical* or *Very Critical*
+
+- The **Asset Overview** dashboard displays *Asset Hostname* as the asset label under **Critical Asset Distribution**.
+
+- Changed the widget name from *MEF Asset* to **Most Essential Function (MEF) Assets** to better describe the asset listing
   
+## Other Enhancements
+
+- The tag names `SampleAsset` and `SampleAlert` have been renamed to `Sample` to simplify alert listing using tags.
```
Comment (cs-dev-chaudhary) [inline release_notes.md]: The tag names `SampleAsset` and `SampleAlert` have been renamed to `Sample` to simplify record listing using tag.

Please change the "alerts" to "record".

File: docs/contents.md [hunk 21]
```diff
@@ -29,67 +21,83 @@ The **OT - Asset Management** solution pack contains the following resources.
...
+| Navigation Menu - Resources |
+
+## Module Schemas
 
-| Name                 | Description                                                                       |
-| :------------------- | :-------------------------------------------------------------------------------- |
-| Full App Permissions | Full App Permissions for module **Asset Resources** and **Asset Change Activity** |
+| Name                  | Description                                                |
+| :-------------------- | :--------------------------------------------------------- |
+| Set Asset Icon        | Introduced new field  `Asset Icon`.                        |
```
Comment (cs-dev-chaudhary) [inline docs/contents.md]: | Asset       | Introduced new field  `Asset Icon` type 'Lookup' with Asset Resources.|

Please change the  "Set Asset Icon"  to "Assets".

File: docs/contents.md [hunk 22]
```diff
@@ -29,67 +21,83 @@ The **OT - Asset Management** solution pack contains the following resources.
...
+|:---------------------------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------|
+| OT - Add Sample Assets                       | Adds 87 different samples of OT Assets (different Purdue Levels, Types, etc.) for testing dashboards, use case playbooks, reports, and other actions. |
+| OT - Add Sample Alerts                       | Adds 12 well-populated samples OT Alerts for testing dashboards, use case playbooks, reports, and other actions.                                      |
+| OT - Stuxnet Attack Scenario                 | Recreates the Stuxnet attack in the form of a series of point alerts received by FortiSOAR from FortiSIEM.                                            |
+| OT - Add Sample Asset Change Activity Record | Generates a sample Asset Change Activity record each, for type *Add New Asset* and *Medium Impact Baseline Change* type of activities.                |
```
Comment (cs-dev-chaudhary) [inline docs/contents.md]: OT - Asset Change Activity Scenario

File: docs/contents.md [hunk 23]
```diff
@@ -29,67 +21,83 @@ The **OT - Asset Management** solution pack contains the following resources.
...
-| OT - Stuxnet Attack Scenario                 | This scenario recreates the Stuxnet attack in form of a series of point alerts received by FortiSOAR from FortISIEM.                                                |
-| OT - Add Sample Asset Change Activity Record | This scenario generates a sample Asset Change Activity record each, for type "Add New Asset" and "Medium Impact Baseline Change" type of activities.                |
+| Name                                         | Description                                                                                                                                           |
+|:---------------------------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------|
+| OT - Add Sample Assets                       | Adds 87 different samples of OT Assets (different Purdue Levels, Types, etc.) for testing dashboards, use case playbooks, reports, and other actions. |
```
Comment (cs-dev-chaudhary) [inline docs/contents.md]: This scenario generates 87 IT/OT assets as per the Purdue model based on various criticality levels, types, and other asset categorizations as sample data.


Same as usage.

File: docs/contents.md [hunk 24]
```diff
@@ -29,67 +21,83 @@ The **OT - Asset Management** solution pack contains the following resources.
...
+| OT - Add Sample Alerts                       | Adds 12 well-populated samples OT Alerts for testing dashboards, use case playbooks, reports, and other actions.                                      |
```
Comment (cs-dev-chaudhary) [inline docs/contents.md]: Description same as usage.

File: docs/contents.md (hunk 22 above)
Comment (cs-dev-chaudhary) [inline docs/contents.md]: *Add New Asset* --> *Add Cyber Asset*

File: docs/setup.md [hunk 25]
```diff
@@ -4,7 +4,7 @@
 # Installation
 
 1. To install a solution pack, click **Content Hub** > **Discover**.
-2. From the list of solution pack that appears, search for and select **OT - Asset Management**.
+2. Search for and select **OT - Asset Management** from the list of solution pack that appears.
```
Comment (cs-dev-chaudhary) [inline docs/setup.md]: search and select.


remove "for".

File: docs/setup.md [hunk 26]
```diff
@@ -19,13 +19,25 @@ The **OT - Asset Management** solution pack depends on the following solution pa
 | MITRE ATT&CK Enrichment Framework | v2.2.0 or later | Required for MITRE Modules and MITRE connector           |
 | Vulnerability Management          | v2.1.0 or later | Required for Vulnerabilities and Scans Module            |
 
+## After Installation
+
+1. Ingest MITRE ATT&CK's **Group**, **Tactics**, **Techniques**, **Sub-techniques**, **Software**, and **Mitigation** data using the *MITRE ATT&CK* connector
+
+> **NOTE**: On alert creation, IOCs are enriched, and the alerts are correlated with Techniques and Mitigation for all records where the MITRE ATT&CK ID is associated with an alert.
```
Comment (cs-ankit-gupta) [inline docs/setup.md]: correlated with MITRE ATT&CK data

File: docs/usage.md [hunk 27]
```diff
@@ -5,90 +5,225 @@
...
+1. Under the FortiSOAR&trade; menu select to open **Simulations**.
 
-2. **OT - Add Sample Alerts**
+2. Select the scenario **OT - Add Sample Assets**.
 
-    - Browse to `Simulations` > `OT - Add Sample Alerts` scenario and click **Simulate Scenario**.
+3. Click **Simulate Scenario**.
 
-    - This scenario adds 12 well populated sample OT Alerts for testing dashboards, use case playbooks, reports and other actions.<br>
+4. To view the added sample assets, under the FortiSOAR&trade; menu select to open **Resources** > **Assets**.
```
Comment (cs-ankit-gupta) [inline docs/usage.md]: Please check the phrase is it making sense

File: docs/usage.md [hunk 28]
```diff
@@ -5,90 +5,225 @@
...
-    - Each alert get associated with assets based on the Source and Destination IP.
+2. Select the scenario **OT - Add Sample Alerts**.
 
-    - **Group**, **Tactics**, **Technique**, **Sub-Technique**, and **Software** get linked to alert under `Correlations` tab of Alert.
-    
-    - **Mitigation** get correlated to alert `Recommended ATT&CK Mitigations` tab.
+3. Click **Simulate Scenario**.
 
-    - Once the alert has been enriched, open the `Stuxnet peer to peer communication attempt` alert and look for a similar alert in the Workspace `Recommendations` tab. 
+4. To view the added sample alerts, under the FortiSOAR&trade; menu select to open **Incident Response** > **Alerts**.
```
Comment (cs-ankit-gupta) [inline docs/usage.md]: Please check the phrase is it making sense

File: docs/usage.md [hunk 29]
```diff
@@ -5,90 +5,225 @@
...
+4. To view the added sample assets, under the FortiSOAR&trade; menu select to open **Resources** > **Assets**.
 
-3. **OT - Stuxnet Attack Scenario**
+![](./res/added-sample-assets.png)
 
-    - Make sure the global variable **Demo mode** is set to `true`. 
+## OT - Add Sample Alerts
 
-    - Browse to `Simulations` > `OT - Stuxnet Attack Scenario` scenario and click **Simulate Scenario**.
+This scenario generates 12 well-populated alerts associated with assets, generated in the previous scenario, as sample data.
```
Comment (cs-ankit-gupta) [inline docs/usage.md]: ......associated with assets, generated with scenario

File: docs/usage.md (hunk 28 above)
Comment (cs-ankit-gupta) [inline docs/usage.md]: Same as above

File: docs/usage.md [hunk 30]
```diff
@@ -5,90 +5,225 @@
...
-- With a given value of due date in `dueDays` two Asset Change Activity entries are made.
-    - Add New Cyber Asset - Siemens S7-400-S621
-    - Medium Impact Baseline Change - Stratix 5950-RW41
+2. Select the scenario **OT - Stuxnet Attack**.
 
-- Assets are linked and can be seen in the Assets & Vulnerabilities tab.
+3. Click **Simulate Scenario**.
 
-- Once the record has been created, execute the **Add Task From Templates** to assign different tasks to different people. 
+4. To view the added sample alerts, under the FortiSOAR&trade; menu select to open **Incident Response** > **Alerts**.
```
Comment (cs-ankit-gupta) [inline docs/usage.md]: Same as above

File: docs/usage.md [hunk 31]
```diff
@@ -5,90 +5,225 @@
...
+    ![](./res/alert-mitre-correlations-tab.png)
+
+- Mitre ATT&CK's suggested *Mitigations* are fetched using the *Technique ID* and are listed under the alert's **Recommended ATT&CK Mitigations** tab.
+
+    ![](./res/alert-mitre-mitigation-tab.png)
+
+<table>
+    <tr>
+        <td><strong>TIP</strong></td>
+        <td>The <strong>Workspace</strong> panel lists alerts similar to <strong>Stuxnet peer-to-peer communication attempt</strong>. To know more about the Workspace panel refer to the platform documentation at <a href="https://docs.fortinet.com/document/fortisoar/7.4.1/user-guide/841633/war-rooms#Workspace_-_Enabling_Communication">Workspace - Enabling Communication</a>.</td>
```
Comment (cs-ankit-gupta) [inline docs/usage.md:104]: link should be -- https://docs.fortinet.com/document/fortisoar/7.4.1/user-guide/841633/war-rooms#Workspace_-_Enabling_Communication

File: release_notes.md [hunk 32]
```diff
@@ -1,55 +1,89 @@
...
+
+    - A widget titled **Recent Alerts On Critical Assets** now comes with pre-defined filter to list assets based on their criticalities &ndash; *Critical* or *Very Critical*
+
+- The **Asset Overview** dashboard displays *Asset Hostname* as the asset label under **Critical Asset Distribution**.
+
+- Changed the widget name from *MEF Asset* to **Most Essential Function (MEF) Assets** to better describe the asset listing
   
+## Other Enhancements
+
+- The tag names `SampleAsset` and `SampleAlert` have been renamed to `Sample` to simplify listing of records using tags.
```
Comment (cs-dev-chaudhary) [inline release_notes.md]: The tag names `SampleAsset` and `SampleAlert` have been renamed to `Sample` for easy identification.

File: docs/usage.md [hunk 33]
```diff
@@ -5,90 +5,225 @@
...
+1. Under the FortiSOAR&trade; menu select to open **Simulations**.
+
+2. Select the scenario **OT - Asset Change Activity Scenario**.
+
+3. Click **Simulate Scenario**.
+
+    <table>
+        <tr>
+            <td><strong>NOTE</strong></td>
+            <td><strong>Scenario - OT - Asset Change Activity</strong> has a variable <code>dueDays</code> (<em>Days required to complete change activity</em>). You can populate these variables with your preferred values in the <em>Configuration</em> step.</td>
```
Comment (cs-ankit-gupta) [inline docs/usage.md]: You can populate "this" variables

File: docs/usage.md [hunk 34]
```diff
@@ -5,90 +5,225 @@
...
+4. Select the **Task Management** tab to open the task management board.
+
+    ![](./res/asset-change-activity-pm-board.png)
+
+5. Edit the open task and mark the status as completed. This moves the task to the *Completed* bucket and creates another task with priority *Medium* under the *Open* bucket. This process repeats until all tasks are moved to *Completed* bucket.
+
+<table>
+    <tr>
+        <td><strong>TIP</strong></td>
+        <td>To add new tasks, click the <strong>Add New Task</strong> button.</td>
```
Comment (cs-ankit-gupta) [inline docs/usage.md:229]: To add standalone new tasks, click.........

**Review by cs-dev-chaudhary — CHANGES_REQUESTED** [review]

File: docs/setup.md (hunk 26 above)
Comment (cs-abhishek-shukla) [reply docs/setup.md]: Fixed

File: docs/usage.md (hunk 27 above)
Comment (cs-abhishek-shukla) [reply docs/usage.md]: Fixed

File: docs/usage.md (hunk 28 above)
Comment (cs-abhishek-shukla) [reply docs/usage.md]: Fixed

File: docs/usage.md (hunk 29 above)
Comment (cs-abhishek-shukla) [reply docs/usage.md]: This scenario generates 12 well-populated alerts associated with assets, generated in the [OT - Add Sample Assets](#ot---add-sample-assets) scenario, as sample data.

File: docs/usage.md (hunk 28 above)
Comment (cs-abhishek-shukla) [reply docs/usage.md]: Fixed

File: docs/usage.md (hunk 30 above)
Comment (cs-abhishek-shukla) [reply docs/usage.md]: Fixed

File: docs/usage.md (hunk 31 above)
Comment (cs-abhishek-shukla) [reply docs/usage.md:104]: Link or link text? Link is same...

File: docs/usage.md (hunk 33 above)
Comment (cs-abhishek-shukla) [reply docs/usage.md]: Fixed

File: docs/usage.md (hunk 34 above)
Comment (cs-abhishek-shukla) [reply docs/usage.md:229]: Not fixed

File: docs/contents.md (hunk 21 above)
Comment (cs-abhishek-shukla) [reply docs/contents.md]: Fixed

File: docs/contents.md (hunk 22 above)
Comment (cs-abhishek-shukla) [reply docs/contents.md]: Fixed

File: docs/contents.md (hunk 23 above)
Comment (cs-abhishek-shukla) [reply docs/contents.md]: Fixed

File: docs/contents.md (hunk 24 above)
Comment (cs-abhishek-shukla) [reply docs/contents.md]: fixed

File: docs/contents.md (hunk 22 above)
Comment (cs-abhishek-shukla) [reply docs/contents.md]: Fixed

File: docs/setup.md (hunk 25 above)
Comment (cs-abhishek-shukla) [reply docs/setup.md]: Fixed

File: release_notes.md (hunk 32 above)
Comment (cs-abhishek-shukla) [reply release_notes.md]: fixed

**Review by cs-dev-chaudhary — APPROVED** [review]

**Pull request merged**